src/compile/scale.ts

```diff
@@ -19,15 +19,10 @@ import {Model} from './model';
 import {smallestUnit} from '../timeunit';
 import {UnitModel} from './unit';
 
-/**
- * Color Ramp's scale for legends.  This scale has to be ordinal so that its
- * legends show a list of numbers.
- */
-export const COLOR_LEGEND = 'color_legend';
-
-// scale used to get labels for binned color scales
-export const COLOR_LEGEND_LABEL = 'color_legend_label';
-
+// scale used to get drive binned legends
+export const BIN_LEGEND_SUFFIX = '_bin_legend';
+// scale used to get labels for legend for binned scales
```

Comment:
/** Scale suffix for binned field's legend labels, which maps a binned field's quantitative values to range strings. */